pxr/imaging/plugin/hdEmbree/renderer.cpp

```diff
@@ -453,6 +534,9 @@ HdEmbreeRenderer::Render(HdRenderThread *renderThread)
 
         // Render by scheduling square tiles of the sample buffer in a parallel
         // for loop.
+#if TBB_INTERFACE_VERSION_MAJOR < 12
```

Comment:
See comment above about scoping; anyway, at the least this should probably move to the top of the function.  I've done this internally but can post it back if you'd like.